src/handler/http/request/dashboards/mod.rs

```diff
@@ -310,43 +282,3 @@ fn get_folder(req: HttpRequest) -> String {
     let query = web::Query::<HashMap<String, String>>::from_query(req.query_string()).unwrap();
     crate::common::utils::http::get_folder(&query)
 }
-
-/// Parses the bytes into a dashboard with the given version.
-fn parse_dashboard(bytes: web::Bytes) -> Result<Dashboard, serde_json::Error> {
-    let d_version: DashboardVersion = json::from_slice(&bytes)?;
-    let version = d_version.version;
-
-    let dash = match version {
-        1 => {
-            let inner: v1::Dashboard = json::from_slice(&bytes)?;
-            inner.into()
-        }
-        2 => {
-            let inner: v2::Dashboard = json::from_slice(&bytes)?;
-            inner.into()
-        }
-        3 => {
-            let inner: v3::Dashboard = json::from_slice(&bytes)?;
-            inner.into()
-        }
-        4 => {
-            let inner: v4::Dashboard = json::from_slice(&bytes)?;
-            inner.into()
-        }
-        _ => {
-            let inner: v5::Dashboard = json::from_slice(&bytes)?;
-            inner.into()
-        }
-    };
-    Ok(dash)
-}
```

Comment:
A version of this logic now lives inside `/http/models` along with the reset of the deserialization definitions